tests/test_cgnsutilities.py

```diff
@@ -138,6 +139,25 @@ def test_refine(self):
         totalCells = self.grid.getTotalCellsNodes()[0]
         self.assertEqual(15120 * 8, totalCells)
 
+    def test_mirror(self):
```

Comment:
This test should be under the `TestReturnFuncs` class because `mirrorGrid` returns a `Grid` object. Also, you shouldn't duplicate code from the function, but create a reference list like in `test_combineGrids`.